README.md

```diff
@@ -0,0 +1,43 @@
+# Aligent PHPStan Pipe
+
+This pipe is used to execute PHPStan over changed files on a PR
+
+## YAML Definition
+
+Add the following your `bitbucket-pipelines.yml` file:
+
+```yaml
+      - step:
+          name: "PHPStan check"
+          script:
+            - pipe: aligent/phpstan-pipe:latest
+              variables:
+                SKIP_DEPENDENCIES: "false"
+                MAGENTO_USER: "USER"
+                MAGENTO_PASS: "PASS"
```

Comment:
Should we be suggesting adding Magento Repo credentials as Repository Secrets instead?